utils/aws/s3.ts

```diff
@@ -0,0 +1,27 @@
+import { ManagedUpload } from "aws-sdk/clients/s3";
+import { s3 } from "aws/index";
+import fs from "fs";
+import path from "path";
+
+/**
+ * Uploads a file to the AWS S3 bucket corresponding to env.AWS_BUCKET_NAME
+ * @param {string} filePath - Path of the file to upload
+ * @returns {object} - S3 upload response data
+ */
+
+const uploadToS3 = (filePath: string): ManagedUpload => {
+    const fileStream = fs.createReadStream(filePath);
+    const uploadParams = {
+        Bucket: process.env.AWS_BUCKET_NAME,
+        Key: path.basename(filePath),
+        Body: fileStream,
+    };
+
+    const res = s3.upload(uploadParams, function (err, data) {
+        console.log(err, data);
+    });
+
+    return res;
```

Comment:
Updated it so that it now returns a promise which the client can work with: https://github.com/uwblueprint/social-diversity-for-children/pull/11/commits/8798d6abefc54c21c27187d32cbade845b641d21